lametro/__init__.py

```diff
@@ -16,14 +17,23 @@ class Lametro(Jurisdiction):
         "events": LametroEventScraper,
     }
 
-    legislative_sessions = []
-    for year in range(2014, 2025):
-        session = {
-            "identifier": "{}".format(year),
-            "start_date": "{}-07-01".format(year),
-            "end_date": "{}-06-30".format(year + 1),
-        }
-        legislative_sessions.append(session)
+    @staticmethod
+    def get_legislative_sessions():
```

Comment:
We want our subclass to have the same signature as [the base `Jurisdiction` class in `pupa`](https://github.com/opencivicdata/pupa/blob/2f7847cb87ed467f7afeec3f51cc704471b679c1/pupa/scrape/jurisdiction.py#L17). How about defining this as a `@property` instead of a `@staticmethod`?